src/internal/m365/collection/drive/group_handler.go

```diff
@@ -175,7 +164,7 @@ func augmentGroupItemInfo(
 		pps = parentPath.String()
 	}
 
-	dii.SharePoint = &details.SharePointInfo{
+	dii.Groups = &details.GroupsInfo{
```

Comment:
I thought about it, but was not sure until now. I'll change it.

Reply:
Hmm, Duplication isn't the best, is it?  Maybe something like this (cross between 1 and 2).

```go
type GroupsSharePointInfo struct {
  DriveID string `json:"driveID, omitempty"`
  DriveName string `json:"driveName, omitempty"`
}

type GroupsInfo struct {
        // old
	Created    time.Time `json:"created,omitempty"`
	ItemName   string    `json:"itemName,omitempty"`
	ItemType   ItemType  `json:"itemType,omitempty"`
	Modified   time.Time `json:"modified,omitempty"`
	Owner      string    `json:"owner,omitempty"`
	ParentPath string    `json:"parentPath,omitempty"`
	Size       int64     `json:"size,omitempty"`
	// new
	ChannelID    string    `json:"channelID,omitempty"`
	ChannelName    string    `json:"channelName,omitempty"`
	SharePointInfo *GroupsSharePointInfo `json:"groupsSharePointInfo,omitempty"`
}
```

Reply:
Now that I think about, I feel like we should just have everything in the root. While it might make sense to separate out `DriveID` and `DriveName`, the only reason why fields like `Owner`, `Modified` etc are in root(even thought they technically belong to SharePoint) are only because they are in the root for other `Info` object.

Reply:
Sounds good!